miden-crypto/src/merkle/mmr/error.rs

```diff
@@ -5,21 +5,30 @@ use thiserror::Error;
 use crate::merkle::MerkleError;
 
 #[derive(Debug, Error)]
+/// Errors that can occur during operations on a Merkle Mountain Range (MMR).
```

Comment:
here and in other places too, we usually put attributes after comments, could we update this?